doc/10-icinga-template-library.md

```diff
@@ -1483,6 +1527,7 @@ Name            | Description
 ----------------|--------------
 users_wgreater  | **Optional.** The user count warning threshold. Defaults to 20.
 users_cgreater  | **Optional.** The user count critical threshold. Defaults to 50.
+users_extra_opts | **Optional.** Read extra plugin options from an ini file.
```

Comment:
Please harmonize the rows' indentations. Yes, previously I wanted https://github.com/Icinga/icinga2/pull/8010#pullrequestreview-413432681 , but let me explain. Given the following table:

```
Lorem ipsum   | dolor sit amet
--------------|---------------
consectetur   | adipiscing
```

... and the following row you’re going to add:

```
elit, sed | do
```

In https://github.com/Icinga/icinga2/pull/8010#pullrequestreview-413432681 I complained about you was doing this:

```patch
@@ -1,3 +1,4 @@
- Lorem ipsum   | dolor sit amet
- --------------|---------------
- consectetur   | adipiscing
+ Lorem ipsum | dolor sit amet
+ ------------|---------------
+ consectetur | adipiscing
+ elit, sed   | do
```

However the following pattern was enough (and you've already corrected it, thanks):

```patch
@@ -1,3 +1,4 @@
  Lorem ipsum   | dolor sit amet
  --------------|---------------
  consectetur   | adipiscing
+ elit, sed     | do
```

So far – so good. But now imagine you'd like to add the following row:

```
elit, sed do eiusmod | tempor
```

According to the above K.I.S.S. scheme you'd do (and did) it like this:

```patch
@@ -1,3 +1,4 @@
  Lorem ipsum   | dolor sit amet
  --------------|---------------
  consectetur   | adipiscing
+ elit, sed do eiusmod | tempor
```

Such are dishamonizing existing tables. **In such cases** please do the following instead:

```patch
@@ -1,3 +1,4 @@
- Lorem ipsum   | dolor sit amet
- --------------|---------------
- consectetur   | adipiscing
+ Lorem ipsum          | dolor sit amet
+ ---------------------|---------------
+ consectetur          | adipiscing
+ elit, sed do eiusmod | tempor
```